docker/scripts/osie.sh

```diff
@@ -130,17 +130,30 @@ if ! [[ -f /statedir/disks-partioned-image-extracted ]]; then
 	mkdir $assetdir
 	echo -e "${GREEN}#### Fetching image (and more) via git ${NC}"
 
-	# config hosts entry so git-lfs assets are pulled through our image cache
-	githost="images.packet.net"
-	images_ip=$(getent hosts $githost | awk '{print $1}')
+	# config hosts entry so git-lfs assets from github and our github-mirror are
+	# pulled through our image cache
+	images_ip=$(getent hosts images.packet.net | awk '{print $1}')
 	cp -a /etc/hosts /etc/hosts.new
-	echo "$images_ip        github-cloud.s3.amazonaws.com" >>/etc/hosts.new && cp -f /etc/hosts.new /etc/hosts
+	{
+		echo "$images_ip        github-cloud.s3.amazonaws.com"
+		echo "$images_ip        github.com"
+		echo "$images_ip        github-mirror.packet.net"
+	} >>/etc/hosts.new
+	# Note: using mv here fails (415 Unsupported Media Type)
```

Comment:
I know why this fails though, its because its bind mounted into the container so we can't move it. So maybe lets make the wording something like:

```
# Note: using mv here fails (415 Unsupported Media Type) because docker sets this up as a bind mount and we can't replace it.
```

Reply:
Ah, I was just reading about this error and wondering why it was showing up in client/server error explanations. Will do.